managed/cmd/pmm-managed/main.go

```diff
@@ -743,6 +744,10 @@ func main() {
 	}
 	defer sqlDB.Close() //nolint:errcheck
 
+	if postgresVersion, err := ver.GetPostgreSQLVersion(ctx, sqlDB); err != nil || postgresVersion.Float() < 14 {
+		l.Panicf("Unsupported PMM-Server PostgreSQL server version: %s. Consider using version 14 or newer", postgresVersion)
```

Comment:
Instead of `Consider ...`, let's use

```
l.Panicf("Unsupported PMM Server PostgreSQL server version: %s. Please upgrade to version 14 or newer", postgresVersion)
```

Reply:
after discussion in slack, it was decided to move to migrateDB and output as an error